app/templates/components/event-card.hbs

```diff
@@ -15,17 +15,21 @@
       </a>
     {{/unless}}
     <div class="main content">
-      <a class="header" href="{{href-to 'public' event.identifier}}">
-        <span>{{event.name}}</span>
-      </a>
+      {{#smart-overflow}}
+        {{#link-to 'public' event.identifier class='ui header'}}
```

Comment:
No specific reason. <a> was not working due to some issue so I tried using link-to but now I have replaced it and it's working fine.